pkg/k8s/apps/crud.go

```diff
@@ -48,6 +48,30 @@ func Get(ctx context.Context, dev *model.Dev, namespace string, c kubernetes.Int
 	return &StatefulSetApp{sfs: sfs}, nil
 }
 
+// Get returns the dev pod for an app and loops until it success
+func Get(ctx context.Context, namespace, appName string, c kubernetes.Interface) (App, error) {
+	d, err := deployments.Get(ctx, appName, namespace, c)
+	if err != nil {
+		return nil, err
+	}
+
+	if err == nil {
```

Comment:
This is already known to be the case and you don't need the if statement. This looks like a bug given there is code after it.